profiles/preview/containerservice/mgmt/containerservice/containerserviceapi/models.go

```diff
@@ -9,7 +9,7 @@
 
 package containerserviceapi
 
-import original "github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2022-02-01/containerservice/containerserviceapi"
+import original "github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2022-03-01/containerservice/containerserviceapi"
```

Comment:
Just curious, why the api version under `profile/preview` is a stable one? I thought this might be `2022-03-02-preview`. But I have to say, I'm not quite sure what the preview profile is for.